.vscode/settings.json

```diff
@@ -4,5 +4,6 @@
     "chpl",
     "inited",
     "rtype"
-  ]
+  ],
+  "git.ignoreLimitWarning": true
```

Comment:
I don't know how you ended up changing this file but I suspect it was accidental. Anyway, if we need to change this file, it should be in another PR. Can you remove the change here? (If necessary you can create a new PR and paste in your new code to make the history simpler).

Reply:
No I did not mean to edit this file, I will remove it now.